.changeset/tough-months-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/compiler": minor
+---
+
+Adds an `walkAsync` utility function that returns a Promise from the tree traversal process. Unlike the existing `walk` function which doesn't provide a way to wait for traversal completion, `walkAsync` allows consumers to `await` the full traversal of the AST
```

Comment:
```suggestion
Adds a `walkAsync` utility function that returns a Promise from the tree traversal process.

Unlike the existing `walk` function which doesn't provide a way to wait for traversal completion, `walkAsync` allows consumers to `await` the full traversal of the AST.
```
Amazing changelog message!  🙌 

Just touched up some grammar, and we usually try to have one standalone sentence to start for easy scannability, then explain details after, so added a paragraph break. What do you think of this?

Reply:
Always happy to accept suggestions, thank you!